Libraries/Components/TextInput/TextInput.js

```diff
@@ -1400,11 +1402,14 @@ function InternalTextInput(props: Props): React.Node {
         ? RCTMultilineTextInputView
         : RCTSinglelineTextInputView;
 
-    const style =
+    let style =
       props.multiline === true
         ? StyleSheet.flatten([styles.multilineInput, props.style])
         : props.style;
 
+    style = flattenStyle(style);
```

Comment:
We should probably remove StyleSheet.flatten from L1407 to avoid calling it twice